aws-cert-converter.py

```diff
@@ -0,0 +1,132 @@
+import argparse
```

Comment:
```suggestion
#!/usr/bin/env python

import argparse
```
The file can be committed with permission +x.